web_src/js/features/notification.js

```diff
@@ -0,0 +1,67 @@
+const {AppSubUrl, csrf} = window.config;
+
+export async function initNotificationsTable() {
+  $('#notification_table .button').click(function () {
+    updateNotification(
+      $(this).data('url'),
+      $(this).data('status'),
+      $(this).data('page'),
+      $(this).data('q'),
+      $(this).data('notification-id')
+    ).then((data) => {
+      $('#notification_div').replaceWith(data);
+      initNotificationsTable();
+      updateNotificationCount();
+    });
+    return false;
+  });
+}
+
+export async function initNotificationCount() {
+  setInterval(() => {
+    updateNotificationCount();
+  }, 5000);
+}
+
+function updateNotificationCount() {
+  $.ajax({
+    type: 'GET',
+    url: `${AppSubUrl}/api/v1/notifications/new`,
+    data: {
+      _csrf: csrf,
+    },
+    success: (data) => {
+      const notificationCount = $('.notification_count');
+      const notificationDependent = $('.notification_dependent');
+      if (data.new === 0) {
+        notificationCount.addClass('hide');
+        notificationDependent.addClass('hide');
+      } else {
+        notificationCount.removeClass('hide');
+        notificationDependent.removeClass('hide');
```

Comment:
I believe the class you want is `hidden`
```suggestion
        notificationCount.addClass('hidden');
        notificationDependent.addClass('hidden');
      } else {
        notificationCount.removeClass('hidden');
        notificationDependent.removeClass('hidden');
```